### PR TITLE
ui(ring-health): fix flaky unit tests

### LINKED ISSUE #822: app unit tests failed: The value provided to the Tabs component is invalid.
```
FAIL src/client/views/ring-health/RingHealth.test.tsx (26.849 s)

Test Suites: 1 failed, 12 passed, 13 total
[2021-06-02T16:23:44Z] Tests:       2 failed, 53 passed, 55 total
```

https://buildkite.com/opstrace/scheduled-main-builds/builds/2407#f7d06d99-a502-4eb1-b1ed-435d67d06e53/500-3300

### PR DESCRIPTION
**Pull request by MoSattler**
Closes #822

# Have you...

* [ ] Discussed your change with a project contributor in an issue yet?
* [x] Added an explanation of what your changes do?
* [x] Written new tests for your changes?
* [x] Thought about which docs need updating?

### CODE REVIEW

File: packages/app/config/jest/setupTests.js [hunk 1]
```diff
@@ -26,4 +29,13 @@ jest.mock("workers", () => {
...
     getOpScriptWorker: () => {}
   };
-});
\ No newline at end of file
+});
+
+jest.setTimeout(15000)
+configure({
+  asyncUtilTimeout: 15000
+})
```
Comment (MoSattler) [inline packages/app/config/jest/setupTests.js:37]: The problem seemed to be that in the CI, the tests tended to timeout - so I increased the timeout time from 5s to 15s.

File: packages/app/config/jest/setupTests.js [hunk 2]
```diff
@@ -26,4 +29,13 @@ jest.mock("workers", () => {
...
+nock.disableNetConnect()
+nock.enableNetConnect(
+  host => host.includes('127.0.0.1') || host.includes('localhost')
+)
```
Comment (MoSattler) [inline packages/app/config/jest/setupTests.js:41]: Here we prevent unit tests calling actual endpoints - unless they're hosted locally (endpoints that are usually mocked in tests).

File: packages/app/src/client/views/ring-health/cortex/RingHealth.test.tsx [hunk 3]
```diff
@@ -93,7 +93,7 @@ describe("CortexRingHealth", () => {
     path,
     endpoint
   ]);
-  xtest.each(tabTestCases)("%s tab", async (tabLabel, tabRoute, tabEndpoint) => {
+  test.each(tabTestCases)("%s tab", async (tabLabel, tabRoute, tabEndpoint) => {
```
Comment (MoSattler) [inline packages/app/src/client/views/ring-health/cortex/RingHealth.test.tsx:96]: reactivating tests.

File: packages/app/src/client/views/ring-health/loki/RingHealth.test.tsx [hunk 4]
```diff
@@ -94,7 +94,7 @@ describe("LokiRingHealth", () => {
     path,
     endpoint
   ]);
-  xtest.each(tabTestCases)(
+  test.each(tabTestCases)(
```
Comment (MoSattler) [inline packages/app/src/client/views/ring-health/loki/RingHealth.test.tsx:97]: reactivating tests.

File: packages/app/config/jest/setupTests.js (hunk 2 above)
Comment (sreis) [reply packages/app/config/jest/setupTests.js:41]: Please add that as a comment in the code.

**Comment by jgehrcke** [comment]
The AWS CI run failed with a known instability (recorded here: https://github.com/opstrace/opstrace/issues/765#issuecomment-864976728).

**Review by jgehrcke — APPROVED** [review]
:rocket: I looked into the CI failure; changes look good -- I assume this is good to go, merging.

File: packages/app/config/jest/setupTests.js (hunk 1 above)
Comment (MatApple) [reply packages/app/config/jest/setupTests.js:37]: @jgehrcke @MoSattler we're still seeing this problem https://buildkite.com/opstrace/scheduled-main-builds/builds/2532#cd42b3ef-4d78-4759-87cd-75c5bba9261e/505-1137 and https://buildkite.com/opstrace/prs/builds/4793#1637cf31-dd2b-4e04-904a-b458ececb5f3/517-627

going to try to bump this timeout an order of magnitude to see if that helps

Comment (MatApple) [reply packages/app/config/jest/setupTests.js:37]: https://github.com/opstrace/opstrace/pull/926

File: packages/app/config/jest/setupTests.js (hunk 2 above)
Comment (MatApple) [reply packages/app/config/jest/setupTests.js:44]: Yes please @MoSattler - great suggestion @sreis

**Pull request merged**